commands/project_eval.go

```diff
@@ -0,0 +1,86 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/phase2/rig/util"
+)
+
+// ProjectEval wrapps the evaluation of project scripts.
+// It mimics command struct except with unexported values.
+type ProjectEval struct {
```

Comment:
Would a better name be `ProjectScript` since it actually wraps the script/execution then you can just have a method of `Run()` and `Result()`?

Reply:
We already have ProjectScript and it's part of how the project_config.go unmarshals the yaml file. We could move ProjectScript to this file, even rename the file to match, and leave project_config.go to reference the struct from the new one.

Reply:
Yeah that sounds good to me, lets move the `ProjectScript` struct here and add these methods to it because they represent the same thing, the config of a project script and the methods to execute it.

Reply:
An alternative is to just call the config struct `Script`  like we did with `Sync` and then rename this to `ProjectScript` just like we have `ProjectSync`